devicetypes/smartthings/zigbee-dual-switch.src/zigbee-dual-switch.groovy

```diff
@@ -0,0 +1,140 @@
+            state "default", label:"", action:"refresh.refresh", icon:"st.secondary.refresh"
+        }
+        main "switch"
+        details(["switch", "refresh"])
+    }
+}
+
+def installed() {
+         createChildDevices()
+         configure()
+}
+
+// Parse incoming device messages to generate events
+def parse(String description) {
+    log.debug "description is $description"
+    Map map = zigbee.getEvent(description)
+    if (map) {
+        if (description?.startsWith('on/off')) {
+            log.debug "receive on/off message without endpoint id"
+        } else {
+            Map descMap = zigbee.parseDescriptionAsMap(description)
+            log.debug "$descMap"
+
+            if (descMap?.clusterId == "0006" && descMap.sourceEndpoint == "01") {
+                def result = map ? createEvent(map) : [:]
+                sendEvent(result)
+             } else if (descMap?.clusterId == "0006" && descMap.sourceEndpoint == "02") {
+                def result = map ? createEvent(map) : [:]
+                def childDevice = childDevices.find {
+			it.deviceNetworkId == "$device.deviceNetworkId-ep2"
+                }
+                childDevice.sendEvent(result)
+	    }
+        }
+    }
+}
+
+private void createChildDevices() {
+       def i = 2
+       addChildDevice("Child Switch", "${device.deviceNetworkId}-ep${i}", null,
+                         [completedSetup: true, label: "${device.displayName.split("1")[-1]} ${i}",
+                         isComponent: false, componentName: "ch$i", componentLabel: "Channel $i"])
+        
+}
+
+private channelNumber(String dni) {
+        dni.split("-ep")[-1] as Integer
+}
+
+def on() {
+     log.debug("on")
+     zigbee.command(0x0006, 0x01, "", [destEndpoint: 0x01])
+}
+
+def off() {
+     log.debug("off")
+     zigbee.command(0x0006, 0x00, "", [destEndpoint: 0x01])
+}
+
+def childOn(String dni) {
+        log.debug("    child on ${dni}")
+        zigbee.command(0x0006, 0x01, "", [destEndpoint: channelNumber(dni)])
+}
+
+def childOff(String dni) {
+        log.debug(" child off ${dni}")
+        zigbee.command(0x0006, 0x00, "", [destEndpoint: channelNumber(dni)])
+}
+
+/**
+* PING is used by Device-Watch in attempt to reach the Device
+* */
+def ping() {
+    return refresh()
+}
+
+def refresh() 
+{    
+    log.debug "refresh broatcast"
+    return zigbee.readAttribute(0x0006, 0x0000, [destEndpoint: 0xFF])
+}
+
+def configure() {
+    log.debug "config"
+    // Device-Watch allows 2 check-in misses from device + ping (plus 2 min lag time)
+    sendEvent(name: "checkInterval", value: 2 * 10 * 60 + 2 * 60, displayed: false, data: [protocol: "zigbee", hubHardwareId: device.hub.hardwareID])
+    
+    return zigbee.configureReporting(0x0006, 0x0000, 0x10, 0, 600, null, [destEndpoint: 0x01]) +
+    		zigbee.configureReporting(0x0006, 0x0000, 0x10, 0, 600, null, [destEndpoint: 0x02]) +
+    		zigbee.readAttribute(0x0006, 0x0000,[destEndpoint: 0x01]) +
```

Comment:
You could replace this and the next line with a single read to the broadcast endpoint.